### ISSUE TITLE
ValueError: too many values to unpack

### ISSUE THREAD

**Issue by Amnesthesia** [milo2012/osintstalker#9]
Crashes with "ValueError: too many values to unpack" when going through all the downloaded data in the end if a user has 'too many' friends or is found in too many photos (I think?).

There are also some problems with encoding when it parses the timeline. Changing str() calls to unicode() worked for me.

**Comment by milo2012** [comment]
Hi Amnesthesia, would you mind to share the lines which you have changed so that I can update the code as well as to test ? 
It would be great if you could share 3 lines before and 3 lines after the 'problem' code. Thanks

**Comment by milo2012** [comment]
Thank you for your time and help on this btw.

**Comment by Amnesthesia** [comment]
Well, I'll attach what I changed but keep in mind I don't actually speak Python, I was just trying to troubleshoot it and googled around a bit and thought that might be a viable solution. But if you treat the strings as bytestrings somewhere, then this might not work well at all :x But it got rid of the encoding issue for me at least (though some letters still turn out strange, but it doesn't cause a problem).

Basically, I just changed all str() calls in: def parseTimeline(html,username) to unicode() calls instead.

**Comment by cndnheat** [comment]
[_] Caching Photo Page: 146781xxxxxx
[_] Caching Photo Page: 157740500xxx
[_] Writing 0 record(s) to database table: photosCommented
list index out of range
xxxxx_friends.htm
[_] Caching Friends Page of: xxxxxxx
Traceback (most recent call last):
  File "fbstalker1.py", line 2030, in <module>
    options(sys.argv)
  File "fbstalker1.py", line 1974, in options
    mainProcess(user)
  File "fbstalker1.py", line 1871, in mainProcess
    dataList = parseFriends(html)
  File "fbstalker1.py", line 1659, in parseFriends
    month,year = value.split(" ")
ValueError: too many values to unpack

Is there a resolution in sight for this?  It causes the script to abort.....

**Comment by Hipasspot** [comment]
Hi milo2012,

Big thanks for making a very interesting script.

Tried the Fbstalker script and got the ValueError: too many values to unpack. My first reaction was that something was wrong with my setup but since others have experienced something similar I just want to ask if somebody have figured out this issue. I don't seem to be able to get around it. I use Kali 32 in virtualbox, I have time to test things if you need anything tested. As I understand it there is an imbalance between the left and right sides of the statement month, year = value.split(" "), but I really don't know what I am talking about. 

[*] Caching Friends Page of: nameoftarget
Traceback (most recent call last):

File "fbstalker.py", line 2030, in <module>
    options(sys.argv)

File "fbstalker.py", line 1974, in options
    mainProcess(user)

File "fbstalker.py", line 1871, in mainProcess
    dataList = parseFriends(html)

File "fbstalker.py", line 1659, in parseFriends

month,year = value.split(" ")
ValueError: too many values to unpack
